nikola/data/samplesite/pages/quickref.rst

```diff
@@ -1089,11 +1089,11 @@
         printed documents, where the link needs to be presented explicitly, for
         example as a footnote.  You can force usage of the call-out form by
         using the
-        "<a href="http://docutils.sourceforge.net/docs/ref/rst/directives.html#target-notes">target-notes</a>"
+        "<a href="https://docutils.sourceforge.io/docs/ref/rst/directives.html#target-notes">target-notes</a>"
         directive.
 
         <p>reStructuredText also provides for <b>embedded URIs</b> (<a
-        href="http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#embedded-uris">details</a>),
+        href="https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#embedded-uris">details</a>),
```

Comment:
Link changed to: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#embedded-uris-and-aliases

(upstream also changed this)

Reply:
Thanks. I didn't do a comprehensive check on the hash components of URLs, and the ones I did check seemed to be fine. I'll update this.

Reply:
I only know because I looked in the repo to check if we have the latest version of the file (and the only new change we were missing was this link).